src/components/ProfilePageComponents/HeaderComponent.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { makeStyles, Typography, Avatar, Box, Button } from "@material-ui/core";
+import { useHistory } from "react-router-dom";
+const useStyles = makeStyles((theme) => ({
+  editbutton: {
+    marginTop: "5px",
+    background: "tansparent",
+    textTransform: "none",
+    padding: "0 16px 0 16px",
+    border: "2px solid #FFFFFF",
+    boxSizing: "border-box",
+    borderRadius: "10px",
```

Comment:
We are now placing useStyles below the functional component to help digging through code better.